docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -64,7 +64,61 @@ The dashboard is no longer accessible, even with the link, until you make it sha
 
 The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
 
-## Supported datasources
+## Email sharing
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the URL.
+
+### Enable email sharing
+
+> **Note:** For Grafana Cloud, contact support to have the feature enabled.
+
+Add the `publicDashboardsEmailSharing` feature toggle to your `custom.ini` file.
+
+```
+[feature_toggles]
+publicDashboardsEmailSharing = true
+```
+
+### Invite a viewer
+
+1. Click the sharing icon to the right of the dashboard title.
+1. Click the **Public dashboard** tab.
+1. Click **Only specified people**.
+1. Enter the email you want to share the public dashboard with.
+1. Click **Invite**.
+1. The recipient(s) will receive an email with a one-time use link. This link must be used within **one hour** or it expires. Once the link is used, viewer has access to the public dashboard for **30 days**.
+
+### Viewers requesting access
+
+If a viewer without access tries to navigate to the public dashboard, they'll be asked to request access by providing their email. They will receive an email with a new one-time use link if the email they provided has already been invited to view the public dashboard and has not been revoked.
```

Comment:
What happens to the request if you haven't already given them access?

Reply:
Nothing happens. They don't get an email or any notification that it hasn't been shared with them. This is intentional though. We don't want unwanted people trying to probe for which emails have access.

Maybe we should show a message after they request access saying "You'll receive an email with a link if you're allowed to view this dashboard", or something along those lines.

Reply:
Does the dashboard owner get a notification of any kind?

Reply:
Not at this point. It would be a nice feature to have, but wasn't in the cards for the V1 of this feature. I imagine it will happen later though.

Reply:
@evictorero @juanicabanas

Reply:
It might be worth saying that explicitly.